glass/core/algorithm.py

```diff
@@ -9,7 +9,7 @@
 import numpy as np
 
 if TYPE_CHECKING:
-    from numpy.typing import ArrayLike
+    from numpy.typing import ArrayLike  # pragma: no cover
```

Comment:
Can we instead ignore it globally? https://coverage.readthedocs.io/en/latest/excluding.html#advanced-exclusion